pom.xml

```diff
@@ -4,11 +4,11 @@
 
 	<groupId>me.figo</groupId>
 	<artifactId>sdk</artifactId>
-	<version>1.2.1</version>
```

Comment:
agreed, maybe we can get this into a shape where nothing gets broken, only deprecated?